Tasks/DownloadSecureFileV1/Tests/L0.ts

```diff
@@ -4,22 +4,67 @@ import path = require('path');
 import * as ttm from 'azure-pipelines-task-lib/mock-test';
 
 describe('DownloadSecureFile Suite', function () {
-    this.timeout(parseInt(process.env.TASK_TEST_TIMEOUT) || 20000);
     before(() => {
     });
 
     after(() => {
     });
 
     it('Defaults: download secure file', (done: MochaDone) => {
```

Comment:
you need to pass `function ()` for the timeout to have any effect.  I believe Mocha does some magic with `this`, which doesn't work with the different semantics for arrow functions.

Reply:
See the doc here: https://mochajs.org/#arrow-functions